src/osd/scrubber/scrub_reservations.cc

```diff
@@ -231,4 +277,4 @@ std::ostream& ReplicaReservations::gen_prefix(
 	 << fmt::format("scrubber::ReplicaReservations:{}: ", fn);
 }
 
-} // namespace Scrub
+}  // namespace Scrub
```

Comment:
Unrelated whitespace change.  Please take the time to check your PRs before submitting them.

Reply:
I actually noticed those changes (introduced by the auto-formatting). I didn't think they were numerous
enough to warrant a separate commit. Will fix.